app/views/tmdb/tv_season.html.erb

```diff
@@ -9,44 +10,50 @@
 </div> <!-- row -->
 
 <div class="row">
-  <% if @series.actors.present? %>
+  <% if @season.cast_members.present? %>
     <h2><%= @season.name %> Cast</h2>
-      <% @series.actors.each do |actor| %>
+      <% @season.cast_members.each do |actor| %>
         <div class="headshot-container">
-        <%= headshot_for(actor) %>
+          <%= headshot_for(actor) %>
           <div class="name-block">
-          <p class="actor"><%= link_to "#{actor.name}", actor_search_path(actor: I18n.transliterate("#{actor.name}"))  %></p>
+            <p class="actor"><%= link_to "#{actor.name}", actor_search_path(actor: I18n.transliterate("#{actor.name}"))  %></p>
             <p>As <%= truncate(actor.character_name, length: 18) %><p>
           </div> <!-- name-block -->
         </div> <!-- headshot-container -->
-        <% end %><!-- actors each do -->
+      <% end %><!-- actors each do -->
   <% end %> <!-- if actors present? -->
 </div> <!-- row -->
 
-<div class="row">
-  <h2><%= @season.name %> Episodes</h2>
-  <table class="table">
-    <tbody>
-      <% @season.episodes.each do |episode| %>
-        <tr>
-          <td>
-            <h2><%= episode.episode_number %>: <%= episode.name %> (<%= episode.air_date.stamp("1/2/2001") %>)</h2>
-            <%= episode.overview %>
-            <p><%= image_tag("http://image.tmdb.org/t/p/w185#{episode.still_path}") if episode.still_path.present? %></p>
-
-            <% if episode.guest_stars.present? %>
-              <h3>Guest Stars</h3>
-              <p>
-              <% episode.guest_stars.each do |guest| %>
-                <%= link_to "#{guest.name}", actor_search_path(actor: "#{guest.name}"), class: 'cast-name-link' %>
-              <% end %><!-- each guest star -->
-              </p>
-            <% end %><!-- if guest star present-->
-          </td>
-        </tr>
-      <% end %>
-    </tbody>
-  </table>
-</div> <!-- row -->
 <hr>
+
```

Comment:
I've moved this whole episode block into two columns and changed out the guest star button links for a `<li>`s with "Actor name as character name"